src/containers/Tablets/Tablets.tsx

```diff
@@ -34,7 +34,9 @@ export function Tablets({
     const [autoRefreshInterval] = useAutoRefreshInterval();
 
     let params: TabletsApiRequestParams = {};
-    const filter = onlyActive ? `(State!=Dead)` : undefined;
+    const filter = onlyActive
+        ? `(State=[Created,ResolveStateStorage,Candidate,BlockBlobStorage,RebuildGraph,WriteZeroEntry,Restored,Discover,Lock,Active,ResolveLeader,Terminating])`
```

Comment:
Number of not active statuses is much less than active. Is it possible to set filter as `State!=[Dead,Deleted,Stopped]`? 

Also it will be good to ensure proper strings with types:
```TS
const activeStatuses: ETabletState = [
    'Created',
    'ResolveStateStorage',
    'Candidate',
    'BlockBlobStorage',
    'RebuildGraph',
    'WriteZeroEntry',
    'Restored',
    'Discover',
    'Lock',
    'Active',
    'ResolveLeader',
    'Terminating'
]

const filter = onlyActive ? `(State=${activeStatuses.join(',')}` : undefined
```